### PR TITLE
Adds alt text for images on landing page

### LINKED ISSUE #2522: Add alt-text for more images
##### Brief description of problem/feature
I've been trying to add alt-text to images as I notice them (which improves the functionality of screen readers on our site). But there are still plenty without alt-text (or it has alt-text but no Spanish translation). We should do a full inventory and fix such cases.

### PR DESCRIPTION
**Pull request by 1kechen**
Resolves #2522 

# Add alt text for images

### CODE REVIEW

File: conf/messages
Comment (misaugstad) [inline conf/messages:6]: Make sure to keep newlines at the ends of files!

**Review by misaugstad — CHANGES_REQUESTED** [review]
I added 2 comments in the code. Could you also look for alt text for any other images that we already have that isn't very helpful and update them (I'm thinking about the collaborator logos). Those logos have alt text like "Makeability_Logo" which isn't very helpful!

**Comment by misaugstad** [comment]
Can you clean up the PR a bit more before I review again? There are translations in the en-NZ file that shouldn't be there, and not all of the new images with alt text have the text in translations files.

Apologies if I looked before you had finished updating the PR. When you're done making updates and the a PR is ready for another review, please post a message on that PR like this so that I know that you're done!

**Comment by misaugstad** [comment]
Also, the messages.zh files has been renamed to messages.zh-TW, so there is a conflict to resolve, unfortunately!

**Comment by 1kechen** [comment]
Ready for review: my edits are split between [#3358](https://github.com/ProjectSidewalk/SidewalkWebpage/pull/3358) and this PR.

[#3358](https://github.com/ProjectSidewalk/SidewalkWebpage/pull/3358) is where I edited index.scala.html to have @Messages in order to translate.

[#2522](https://github.com/ProjectSidewalk/SidewalkWebpage/pull/3358) (here) is where I edited all of the conf/messages.[LANGUAGE CODE] to hold the translations.

**Comment by misaugstad** [comment]
Can you resolve merge conflicts before I review? I suppose I should wait until after the People Nudge logo PR has been merged as well, but merge conflicts are going to need to be fixed at some point anyway!

**Comment by misaugstad** [comment]
It shouldn't be too bad. Really all that we did was rename "messages.zh" to "messages.zh-TW" and Github doesn't know how to handle that automatically.

File: app/views/index.scala.html
Comment (misaugstad) [inline app/views/index.scala.html:202]: You should just use the actual translations here. I know that you did that in your other PR, but you've ultimately set it up so that I have to merge both of your PRs at once, otherwise neither can be merged. 

You should make each PR as separate from each other as possible, and we can deal with merge conflicts in the future if it ends up being an issue.

File: conf/messages.en
Comment (misaugstad) [inline conf/messages.en:347]: This should probably say something like "white clouds partially covering the image above, with statistics shown on top of the clouds"

**Review by misaugstad — CHANGES_REQUESTED** [review]
We should probably add alt text for these logos in the footer as well!
Screenshot from 2023-08-29 15-36-07

**Review by misaugstad — APPROVED** [review]
Looks good! I did make a couple changes myself, listed below:
* As per [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-guide), make sure that your files end with newlines. When you added to the messages files, you removed the blank lines at the ends of files.
* We only need to include a "en-NZ" translation if it's different from "en". As such, I was able to remove all the translations you added to that file.

**Pull request merged**
